code/game/machinery/rechargestation.dm

```diff
@@ -244,45 +244,49 @@
 				// ^ makes single list of active (R.contents) and inactive modules (R.module.modules)
 				for(var/obj/O in um)
 					// Engineering
-					if(istype(O,/obj/item/stack/cable_coil))
-						if(O:amount < 50)
-							O:amount += 2
-						if(O:amount > 50)
-							O:amount = 50
+					if(istype(O,/obj/item/stack))

```

Comment:
This really isn't a good idea. It makes any stack regenerate if you have 1 sheet, including reinforced plasma glass.

Better to remove this whole section and make cable coils able to be produced from the mommi synthesizer.